Server/Resources.xcassets/springboard-alerts/springboard-alerts-pt_BR.dataset/alerts.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "title": "“%1$@” Deseja Ter Acesso às Contas do %2$@",
```

Comment:
What does `%1$@` and `%2$@` mean?
AFAIK, the app name in the title should be replaced with `%@`. Are these two parameters?